usaspending_api/common/helpers/s3_helpers.py

```diff
@@ -0,0 +1,52 @@
+import boto3
+import io
+import logging
+import re
+
+from typing import Optional, List
+
+from usaspending_api import settings
+
+logger = logging.getLogger("script")
+
+
+def access_s3_object_list(
```

Comment:
Are we doing away with the ability to be able to run loaders locally?  I could have sworn the old FPDS loader used to look in my `csv_downloads` directory for delete files using the `IS_LOCAL` setting.

Reply:
Yes it did. 
1. The local file functionality was poorly designed and implemented. If has a benefit should  remain it needs to be refactored.
2. I am trying to make the FABS and FPDS loaders as shared as possible. If there is a benefit to having local files for deletes it should be added to FABS too.

Reply:
I just wanted to make sure it was a conscious decision.  It did make it a little easier to test but I agree that that was clunky code.  And FABS is a whole other thing since it writes.  That makes it extra scary to test against S3.